backend/src/routes/api/nb-events/index.ts

```diff
@@ -0,0 +1,17 @@
+import { FastifyInstance } from 'fastify';
+import { getNotebookEvents } from './eventUtils';
+
+export default async (fastify: FastifyInstance): Promise<void> => {
+  fastify.get('/:notebookName', async (request, reply) => {
```

Comment:
type `request` to `request: FastifyRequest<{ Params: { notebookName: string } }>` so we can get rid of `as...` below.